code/crates/network/src/lib.rs

```diff
@@ -179,28 +180,45 @@ pub async fn spawn(
     keypair: Keypair,
     config: Config,
     registry: SharedRegistry,
+    protocol_names: ProtocolNames,
```

Comment:
Same here, the protocol names are already part of the config.

Reply:
https://github.com/circlefin/malachite/pull/1179/commits/da512eceb0740b42365fee5a983ee74b14940950